src/eko/evolution_operator/operator_matrix_element.py

```diff
@@ -20,6 +21,33 @@
 logger = logging.getLogger(__name__)
 
 
+class BackwardsMethods(enum.IntEnum):
```

Comment:
maybe `MatchingMethods` would be better as `BackwardsMethods.FORWRD` is a bit of a contraddciton

Reply:
agreed - done in c88627f7